docs/data/date-pickers/overview/overview.md

```diff
@@ -17,6 +17,7 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/datepick
 
```

Comment:
Not relevant to these changes, but there's an issue with the main demo overflowing on small screen sizes:

<img width="916" alt="Screenshot 2024-11-13 at 08 23 31" src="https://github.com/user-attachments/assets/aa96c05e-31ea-4f84-84ff-6bf2a9a38084">

This seems to solve it:

```diff
diff --git a/docs/src/modules/components/overview/MainDemo.tsx b/docs/src/modules/components/overview/MainDemo.tsx
index 279e80189..52e697ed6 100644
--- a/docs/src/modules/components/overview/MainDemo.tsx
+++ b/docs/src/modules/components/overview/MainDemo.tsx
@@ -89,7 +89,7 @@ export default function MainDemo() {
       <Paper component="div" variant="outlined" sx={{ mb: 8 }}>
         <ThemeProvider theme={showCustomTheme ? customTheme : theme}>
           <Stack sx={{ p: 1, width: '100%' }} direction="row" spacing={1}>
-            <Stack spacing={1} className="left-panel" sx={{ flexGrow: 1 }}>
+            <Stack spacing={1} className="left-panel" sx={{ flexGrow: 1, minWidth: 0 }}>
               <Stack
                 spacing={1}
                 direction={isMobile ? 'column' : 'row'}
```